packages/forms/src/components/ui/radio/RadioButton.tsx

```diff
@@ -1,163 +1,48 @@
 import { ChangeEvent, useCallback, useRef } from "react";
 import { FullOnChangeProps } from "../types";
-import { defaultRadioButtonTheme, RadioButtonTheme } from "./RadioButtonTheme";
+import styles from "./RadioButton.module.css";
+
+export type RadioButtonSize = "standard" | "small";
 
 export interface RadioButtonProps
-  extends FullOnChangeProps<boolean, ChangeEvent<HTMLInputElement>>,
-    InputProps<HTMLButtonElement> {
-  disabled?: boolean;
-  theme?: RadioButtonTheme;
+  extends FullOnChangeProps<string, ChangeEvent<HTMLInputElement>>,
+    Omit<InputProps<HTMLButtonElement>, "size"> {
+  size?: RadioButtonSize;
 }
 
-const InvisibleRadioButton = styled.input`
-  top: 0;
-  left: 0;
-  width: 100%;
-  cursor: inherit;
-  height: 100%;
-  margin: 0;
-  opacity: 0;
-  padding: 0;
-  position: absolute;
-`;
-
 export const RadioButton: React.FC<RadioButtonProps> = ({
-  className,
-  disabled = false,
   onChange,
   onValueChange,
-  theme = defaultRadioButtonTheme,
-  value = false,
   inputRef,
-  name,
-  wrapperRef
+  size = "standard",
+  ...inputProps
 }) => {
-  const { colors } = useThemeFields(
-    {
-      colors: {
-        iconColorDisabled: theme.iconColorDisabled,
-        iconColor: theme.iconColor,
-        iconColorNotChecked: theme.iconColorNotChecked,
-        iconColorNotCheckedHover: theme.iconColorNotCheckedHover
-      }
-    },
-    [theme]
-  );
-
   const innerInputRef = useRef(null);
 
   const inputRefToUse = inputRef || innerInputRef;
 
-  const mouseIsOver = useMouseIsOver(inputRefToUse);
-
-  const icon = getIcon(value, disabled, mouseIsOver, theme);
-
-  const onClickHandler = useCallback(
-    ev => {
+  const handleInputChange = useCallback(
+    (ev: ChangeEvent<HTMLInputElement>) => {
       if (onChange) {
         onChange(ev);
       }
       if (onValueChange) {
-        onValueChange(true);
+        onValueChange(ev.target.value);
       }
     },
     [onChange, onValueChange]
   );
 
-  const handleInputChange = useCallback(
-    (ev: ChangeEvent<HTMLInputElement>) => {
-      if (!disabled) {
-        if (onChange) {
-          onChange(ev);
-        }
-        if (onValueChange) {
-          onValueChange(ev.target.checked);
-        }
-      }
-    },
-    [disabled, onChange, onValueChange]
-  );
-
   return (
-    <Box position={"relative"}>
-      <Clickable
-        onClick={disabled ? undefined : onClickHandler}
-        innerRef={wrapperRef}
-      >
-        <Icon
-          color={getIconColor(value, disabled, mouseIsOver, colors)}
-          icon={icon}
-          hoverColor={"red"}
-          size={theme.iconSize}
-        />
-        <InvisibleRadioButton
-          disabled={disabled}
-          checked={value}
-          ref={inputRefToUse}
-          onChange={handleInputChange}
-          type={"radio"}
-          name={name}
-          className={className}
-        />
-      </Clickable>
-    </Box>
+    <input
+      type={"radio"}
+      role={"radio"}
```

Comment:
```suggestion
```

Reply:
Remove since the linter complains about it? 😊

Reply:
Thanks :)